tests/Message/FetchTransactionRequestTest.php

```diff
@@ -253,4 +255,107 @@ public function testSendByReferenceFailure()
 
         $this->assertNull($response->getTransaction());
     }
+
+    /**
+     * @return void
+     */
+    public function testSendPayPalSuccess()
+    {
+        $this->setMockSoapResponse('FetchTransactionPayPalSuccess.xml', array(
```

Comment:
can you name this FetchPayPalTransactionSuccess? That's the way other things are named